code/game/objects/items/weapons/grenades/chem_grenade.dm

```diff
@@ -533,3 +533,16 @@ obj/item/weapon/grenade/chem_grenade/exgrenade/attackby(obj/item/weapon/W as obj
 	icon_state = null
 	volume = 1000
 	flags = FPRINT  | OPENCONTAINER | NOREACT
+
+/obj/item/weapon/grenade/chem_grenade/timer  //grenade casing with a 5 second timer+ignitor installed. no beakers.
+											// can be configured via a right click verb. can be done before being assembled and after being assembled
+	name = "grenade casing (timer)"
+	desc = "A hand made chemical grenade. This one seems to have a dial on the top"
+	starting_materials = list(MAT_IRON = 4750, MAT_GLASS = 100)
+	stage = 1		//these are marked as 1 because inserting any parts into an empty grenade casing raises the value from 0 to 1
```

Comment:
use defines for these

Reply:
elaborate, what values should be defined?

Reply:
stage and path, create defines for them at the chem_grenade level

I know this isn't really your fault for implementing it this way, but it will be better readability for the future and not much effort on your part to improve the oldcoding on this

Reply:
i attempted to do this to more weirdness, Ill try it again but no promises i could adjust a system like this without creating more bugs and problems

Reply:
Are you misunderstanding what I'm asking? This should not be a significant change, just create some defines and use those instead of the nondescript number 1. This doesn't require any system adjustments, it's cosmetic.

Just remember the define needs to be made before the first instance of its use, which is why we put most defines in the files at the top.

Reply:
something along the lines of 
```
#define GRENADE_STAGE_EMPTY 0
#define GRENADE_STAGE_ASSEMBLY_INSERTED 1
#define GRENADE_STAGE_COMPLETE 2
```
and then 
```
stage = GRENADE_STAGE_ASSEMBLY_INSERTED
```
is what i think kurfursten wanted

Reply:
Yes, as simple as that. Defines go at the top or in a defines file near the top of the whole project.